src/parser/content-evaluator-module.ts

```diff
@@ -91,49 +112,41 @@ export class ContentEvaluatorModule implements Module {
     return commentsWithScore;
   }
 
-  _relevancesForCommentTypes(): { [k: string]: number } {
-    const relevances: { [k: string]: number } = {};
-    this._configuration.multipliers.forEach((multiplier) => {
-      let commentType = 0;
-      multiplier.targets.forEach((target) => {
-        commentType |= CommentType[target as keyof typeof CommentType];
-      });
-
-      if (multiplier.relevance !== undefined) {
-        relevances[commentType as unknown as string] = multiplier.relevance;
-      }
-    });
-    return relevances;
-  }
-
   async _evaluateComments(
     specification: string,
     comments: { id: number; comment: string }[]
   ): Promise<{ [k: string]: number }> {
     const prompt = this._generatePrompt(specification, comments);
 
-    try {
-      const response: OpenAI.Chat.ChatCompletion = await this._openAi.chat.completions.create({
-        model: "gpt-4o",
-        response_format: { type: "json_object" },
-        messages: [
-          {
-            role: "system",
-            content: prompt,
-          },
-        ],
-        temperature: 1,
-        max_tokens: 128,
-        top_p: 1,
-        frequency_penalty: 0,
-        presence_penalty: 0,
-      });
-
-      const rawResponse = String(response.choices[0].message.content);
-      return JSON.parse(rawResponse) as { [k: string]: number };
-    } catch (error) {
-      console.error(`Failed to evaluate comment`, error);
-      return {};
+    const response: OpenAI.Chat.ChatCompletion = await this._openAi.chat.completions.create({
+      model: "gpt-4o",
+      response_format: { type: "json_object" },
+      messages: [
+        {
+          role: "system",
+          content: prompt,
+        },
+      ],
+      temperature: 1,
+      max_tokens: 128,
+      top_p: 1,
+      frequency_penalty: 0,
+      presence_penalty: 0,
+    });
+
+    const rawResponse = String(response.choices[0].message.content);
+    logger.info(`OpenAI raw response: ${rawResponse}`);
+
+    const jsonResponse = JSON.parse(rawResponse);
+
+    const responseType = Type.Record(Type.String(), Type.Number({ minimum: 0, maximum: 1 }));
```

Comment:
usually typebox schemas are in types folder

Reply:
resolved by https://github.com/ubiquibot/conversation-rewards/pull/55/commits/ac4b164ba2bedd83ff3d17774cebb5b4e63aa5ff
QA [before](https://github.com/EresDevOrg/ubiquibot-issues/issues/10#issuecomment-2283396407) and [after](https://github.com/EresDevOrg/ubiquibot-issues/issues/10#issuecomment-2283536986) the change.